pypowsybl/network.py

```diff
@@ -4124,6 +4124,228 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def create_load_bay(self, busbar_section_id: str, load_position_order: int, df: _DataFrame,
+                        load_direction: str = "BOTTOM", throw_exception: bool = False,
+                        reporter: _Reporter = None, **kwargs: _ArrayLike) -> None:
+        """
+        Creates a load, connects it to the network and creates the topology. The voltage level containing the
+        busbar section should be described in node/breaker topology. The load is connected to the busbar with a
+        breaker and a closed disconnector. If the network has position extensions, the load will also be connected
+        to every parallel busbar section with an open disconnector.
+
+        Args:
+            busbar_section_id: id of the busbar section to which the injection will be connected with a closed
+            disconnector.
+            load_position_order: the order of the load, will fill the ConnectablePosition extension
+            df: the dataframe containing the attributes of the load.
+            load_direction: the direction of the load, will fill the ConnectablePosition extension, default
+            is BOTTOM.
+            throw_exception: whether the calculation should throw exceptions. In any case, errors will be logged.
```

Comment:
The goal of this boolean is to allow the use of CreateFeederBay with no failure. Is there another option in Python that would replace it?

Reply:
I mean that the user could simply do:
```python
try:
    pp.create_feeder_bay(...)
except pp.PyPowsyblError:
    # log or something else
```

But actually the option can be useful if we allow to create multiple feeders with one call (with a dataframe):
this way it will not stop on first failure.

Let's keep it and rename it raise_exception ? (in python they say raise instead of throw :) )